pkg-r/README.md

```diff
@@ -5,9 +5,11 @@
 [![CRAN status](https://www.r-pkg.org/badges/version/shinychat)](https://CRAN.R-project.org/package=shinychat)
 <!-- badges: end -->
 
-Chat UI component for [Shiny for R](https://shiny.posit.co/).
+**shinychat** is a chat UI component for [Shiny for R](https://shiny.posit.co/).
 
-(For [Shiny for Python](https://shiny.posit.co/py/), see [ui.Chat](https://shiny.posit.co/py/components/display-messages/chat/).)
+It’s also available for [Shiny for Python](https://posit-dev.github.io/shinychat/py/index.html).
```

Comment:
It feels slightly off putting to point to another language so early in the description.

```suggestion
```